content/en/docs/concepts/architecture/nodes.md

```diff
@@ -173,18 +173,27 @@ to be unreachable. (The default timeouts are 40s to start reporting
 ConditionUnknown and 5m after that to start evicting pods.) The node controller
 checks the state of each node every `--node-monitor-period` seconds.
 
-In versions of Kubernetes prior to 1.13, NodeStatus is the heartbeat from the
-node. Node lease feature is enabled by default since 1.14 as a beta feature
-(feature gate `NodeLease`, [KEP-0009](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/0009-node-heartbeat.md)).
-When node lease feature is enabled, each node has an associated `Lease` object in
-`kube-node-lease` namespace that is renewed by the node periodically, and both
-NodeStatus and node lease are treated as heartbeats from the node. Node leases
-are renewed frequently while NodeStatus is reported from node to master only
-when there is some change or enough time has passed (default is 1 minute, which
-is longer than the default timeout of 40 seconds for unreachable nodes). Since
-node lease is much more lightweight than NodeStatus, this feature makes node
-heartbeat significantly cheaper from both scalability and performance
-perspectives.
+#### Heartbeats
+
+Heartbeats, sent by Kubernetes nodes, help determine the availability of a node.
+There are two forms of heartbeats: updates of `NodeStatus` and the
+[Lease object](/docs/reference/generated/kubernetes-api/{{< latest-version >}}/#lease-v1-coordination-k8s-io).
+Each Node has an associated `Lease` object in the `kube-node-lease`
```

Comment:
These changes look good. If you are cleaning up nits ...
- There is one more Lease to clean up the backticks on line 181.
- Also, I _think_ command tools (kubelet, kubectl) typically are quoted with backticks, throughout the docs.
  For example: https://kubernetes.io/docs/tasks/administer-cluster/out-of-resource/

Reply:
`kubectl` is 100% a command line tool. `kubelet`, whilst also an executable,  isn't a tool that people _usually_ run in a command line, at least not during typical cluster operation. I see `kubelet` fitting into the same slot as `kube-scheduler`, `kube-controller-manager`, and `kube-proxy`.

(I've used backticks for those names in this very comment but in the docs I typically wouldn't).

Reply:
Fixed the Lease.

I agree with @sftim that kubelet is in the same category as kube-scheduler, etc.
[in this file kubelet is not in backsticks anywhere]

Reply:
Hello @wojtek-t . It feels like these changes should be ready 😀 . Okay to override these comments, but if you have the energy for one more change, I'd suggest splitting/adjusting lines 183 - 184.

Lease is a lightweight resource, which improves the performance of the node heartbeats as the cluster scales.
OR
Lease is a smaller resource than the Node, which improves the performance of the node heartbeats as the cluster scales.

Line 186: The kubelet is responsible for creating and updating the `NodeStatus` and a Lease object.

Reply:
Done.